zebra-state/src/service/queued_blocks.rs

```diff
@@ -217,6 +217,12 @@ impl QueuedBlocks {
 
         self.blocks.drain()
     }
+
+    /// Returns true if QueuedBlocks contains a block with the given hash.
+    /// Returns false otherwise.
+    pub fn contains(&self, hash: &block::Hash) -> bool {
```

Comment:
To avoid confusion with UTXO transaction hashes.

```suggestion
    pub fn contains_block_hash(&self, hash: &block::Hash) -> bool {
```